packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/extension.tsp

```diff
@@ -0,0 +1,173 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+model ExtensionsResource is ExtensionResource<ExtensionsResourceProperties> {
+  ...ResourceNameParameter<ExtensionsResource>;
+}
+
+/** ExtensionsResource properties */
+model ExtensionsResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+@armResourceOperations
+interface ExtensionsResources {
+  @scenario
+  @scenarioDoc("""
+    Resource GET operation.
+    Expected path: /<resourceUri>/providers/Azure.ResourceManager.Models.Resources/extensionsResources/extension
```

Comment:
you need to give the full test path that emitter could know how to create this resource

Reply:
Fixed by commit [9103aac](https://github.com/Azure/cadl-ranch/pull/701/commits/9103aac3c596fbcec47a70c86aec3c1c60dd5628)